process/project-lifecycle.md

```diff
@@ -159,6 +159,16 @@ When voting on a proposal to archive a project, TAC members may consider:
 * If the project's inactivity or inconsistent maintenance presents a user security risk.
 * If the project's design or approach is no longer a recommended best practice.
 
+#### Considerations when Archiving a TI that has sub-efforts embedded within ####
+
+1.) Overseeing TI will be moving to "Archived" status
+2.) Whether subgroups/-TIs (SIG or project) are still active and desire to continue collaborating
+3.) Subgroup reviews active TIs to see if there is alignment of vision and purpose (https://openssf.org/community/openssf-working-groups/)
+4.) Subgroup meets with TI leader and group to discuss adoption
+5.) New TI agrees to become new oversight group for subgroup
+6.) PRs filed in TI and TAC repos to note change of status
+7.) Archiving TI has PR filed noting new status
```

Comment:
```suggestion
1. Overseeing TI will be moving to "Archived" status
2. Whether subgroups/-TIs (SIG or project) are still active and desire to continue collaborating
3. Subgroup reviews active TIs to see if there is alignment of vision and purpose (https://openssf.org/community/openssf-working-groups/)
4. Subgroup meets with TI leader and group to discuss adoption
5. New TI agrees to become new oversight group for subgroup
6. PRs filed in TI and TAC repos to note change of status
7. Archiving TI has PR filed noting new status
```

Minor: GitHub's Markdown renderer will smoosh all these items onto one line if the format `1.)` is used, but `1.` will create a list 🤷

Reply:
In addition to removing the `)`, every item can be `1.` and the markdown render will automatically increment. That makes future PRs easier if anything ever gets added or removed from the middle of the list.